info/_HELP-create-build.txt

```diff
@@ -99,7 +99,7 @@ see "build\build.eclipse.4.23.nl.repo\How to update this p2 repo.txt"
 Create Dojo files, when web-UI in net.tourbook.web is modified
 ################################################################
 
-   - Set "IS_DEBUG = false" in net.tourbook.web.WEB 
+   - Set "IS_DEBUG = false" in net.tourbook.web/WEB.java 
```

Comment:
I'm mostly using the action _Copy Qualified Name_ in the context menu, when I need a Java name, this returned `net.tourbook.web.WEB`